src/overlays/actors/ovl_En_Thiefbird/z_en_thiefbird.c

```diff
@@ -220,21 +220,22 @@ s32 func_80C10B0C(EnThiefbird* this, GlobalContext* globalCtx) {
     static Gfx* D_80C13680[] = { D_06004348, D_06004B88, D_060055E0 };
     s32 isItemFound = false;
     s32 phi_a3 = 0;
-    s32 i = 18;
-    s32 phi_t0_3;
-    s16 sp1E = 0;
+    s32 slotId = SLOT_BOTTLE_1;
+    s32 itemId1;
+    s16 itemId2 = ITEM_OCARINA;
```

Comment:
I'm not sure a define like that should be local to this actor as setting it to 0 likely has significance in `func_801149A0` instead of within this file. I think I'd prefer to replace `ITEM_OCARINA` to 0 and wait until `func_801149A0` is decomped before overlapping both `ITEM_OCARINA` and `ITEM_NONE`, thoughts?

Reply:
I'm fine with replacing it with a 0, mainly just removing the `ITEM_OCARINA`. However I think a local define works since `func_801149A0` never gets called with `itemId2=0`, since `isItemFound` stays false.